op/handlers.go

```diff
@@ -66,6 +66,9 @@ func NewHandler(os *Status) http.Handler {
 	m.Handle("/__/ready", newReadyHandler(os))
 	m.Handle("/__/metrics", promhttp.Handler())
 
+	// Overload default mux in order to stop pprof binding handlers to it
+	http.DefaultServeMux = http.NewServeMux()
```

Comment:
I am thinking about what this will do to applications that are currently relying on pprof being in the default mux?  I guess they lose their pprof and then discover that it's just moved in practice and they can remove their old pprof code. It's not the end of the world I suppose.

Reply:
correct, will document this in the README